strategies/failed/pom.xml

```diff
@@ -33,6 +33,11 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.arquillian.smart.testing</groupId>
+      <artifactId>junit-test-result-parser</artifactId>
+      <version>${project.version}</version>
```

Comment:
I would say if we have SPI but do not open it up so we can plug different implementations this really shouldn't be here. For the time being `surefire-provider` might be ok, but this is not the ideal solution either.